Source/ZoomNet/Resources/Phone.cs

```diff
@@ -94,6 +96,36 @@ public Task<PaginatedResponseWithToken<PhoneUser>> ListPhoneUsersAsync(
 				.AsPaginatedResponseWithToken<PhoneUser>("users");
 		}
 
+		/// <inheritdoc/>
+		public Task UpdateCallHandlingSettingsAsync(
+			string extensionId,
+			CallHandlingSettingType settingType,
+			CallHandlingSubsettingsBase subsettings,
+			CancellationToken cancellationToken)
+		{
+			if (!settingType.TryToEnumString(out string settingTypeString))
```

Comment:
Is there a reason to perform validation here? Our `StringEnumConverter` already validates enum values so you should be able to simply :

```csharp
var data = new JsonObject
{
    { "settings", subsettings },
    { "sub_setting_type", settingType } <-- this enum value will be automatically converter to a string by our `StringEnumConverter`
};
```

I know that I haven't been 100% consistent. There are some instances where I have manually converted enum values and some instances where I relied on the StringEnumConverter like these two examples:
```csharp
{ "action", createType.ToEnumString() }, <-- manually convert the enum value to a string
{ "type", type }, <-- rely on the StringEnumConverter to convert the enum value to a string
```

StringEnumConverter tries to convert the enum value to a string and, if unable to do so, falls back on using the integer value of the enum value. That's to only difference that I can think of between manually converting versus relying on StringEnumConverter.
```csharp
else if (value.TryToEnumString(out var stringValue))
{
	writer.WriteStringValue(stringValue);
}
else
{
	writer.WriteNumberValue(Convert.ToInt32(value));
}
```

All this to say that we should strive to standardize how we serialize enum values and, as a rule of thumb, we should rely on StringEnumConverter. But maybe in this specific case there's a reason you decided to manually convert the value to a string and perform validation, I'm curious.